README.md

```diff
@@ -24,6 +24,14 @@ git clone https://github.com/gjonesme/MMM-WeatherAlerts
 cd MMM-WeatherAlerts && npm install
 ```
 
+# Update
+
+```
```

Comment:
I recommend adding `bash` after the backticks. Checkout [MD040](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md040.md).

Sorry if I'm being too picky! :smile:

Reply:
Done